graph/def.proto

```diff
@@ -116,3 +116,23 @@ message DefDoc {
     // Data is the actual documentation text.
     string data = 2 [(gogoproto.jsontag) = "Data"];
 };
+
+// DefFormatStrings contains the various def format strings.
```

Comment:
Mention something like: for more information, see the godoc in the srclib/graph package.